### ISSUE TITLE
fix gantz setup

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#5709]
In the last round of testing on gantz there were some issues that look like setup problems that I need to fix:
```
        FAILED  connections/gevent_connection.py2.6
            ImportError: No module named gevent
        FAILED  connections/http.js0.10
            ImportError: No module named twisted.internet
        TIMEOUT connections/http.py2.6
            ImportError: No module named twisted.internet
        FAILED  connections/http.rb2.0
            ImportError: No module named twisted.internet
        FAILED  connections/twisted_connection.py2.6
            ImportError: No module named twisted.internet
```

This should be easy, this is just the reminder to myself to do it.

**Comment by larkost** [comment]
Also, `tested` needs to be fixed on `newton` for Python 3.3 and 3.4 (`ImportError: cannot import name cookiejar`).

**Comment by larkost** [comment]
I have fixed these issues.
